File: solutions/2025-dec/18/checkerboard.js
function createBoard(dimensions) {
    const ans = [];
    for (let i = 0; i < dimensions[0]; i++) {
        ans.push([]);
        for (let j = 0; j < dimensions[1]; j++) {
            if (((j % 2) + (i % 2)) % 2 == 0) ans[i].push("X");
            else ans[i].push("O");
        }
    }
    return ans;
}

File: solutions/2025-dec/18/checkerboard.py
def create_board(dimensions):
    ans = []
    for i in range(dimensions[0]):
        ans.append([])
        for j in range(dimensions[1]):
            if (j % 2 + i % 2) % 2 == 0:
                ans[i].append("X")
            else:
                ans[i].append("O")
    return ans
